packages/block-library/src/navigation-menu/index.php

```diff
@@ -15,20 +15,69 @@
  * @return string Returns the post content with the legacy widget added.
  */
 function render_block_navigation_menu( $attributes, $content, $block ) {
-	return '<nav class="wp-block-navigation-menu">' . build_navigation_menu_html( $block ) . '</nav>';
+	// Add CSS classes and inline styles.
+	$bg_color_css_classes = '';
+	if ( array_key_exists('backgroundColor', $attributes ) ) {
+		$bg_color_css_classes .= ' has-background-color';
+	}
+	if ( array_key_exists('backgroundColorCSSClass', $attributes ) ) {
+		$bg_color_css_classes .= " {$attributes['backgroundColorCSSClass']}";
+	}
+	$bg_color_css_classes = trim( $bg_color_css_classes );
+
+	$text_color_css_classes = '';
+	if ( array_key_exists('textColor', $attributes ) ) {
+		$text_color_css_classes .= ' has-text-color';
+	}
+
+	if ( array_key_exists('textColorCSSClass', $attributes ) ) {
+		$text_color_css_classes .= " ${attributes['textColorCSSClass']}";
```

Comment:
`${attributes` will need to become `{$attributes` for it to work

Reply:
Actually it does, but let me update it.